package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "license": "UNLICENSED",
   "scripts": {
-    "build-prisma": "prisma-import -s src/**/*.prisma -o prisma/schema.prisma --force && prisma generate",
+    "build-prisma": "prisma-import -s 'src/**/*.prisma' -o prisma/schema.prisma --force && prisma generate",
```

Comment:
this does not work on Windows

Reply:
however, without this, it does not work on linux

Reply:
So it's better to use \", and please change the bottom one as well.